src/partials/Toast.svelte

```diff
@@ -75,10 +82,16 @@
       currentOffset = 0
     }, 200)
   }
+
+  const timeInterval = setInterval(() => timeLeft > 0 && timeLeft--, 1000)
+
+  onDestroy(() => {
+    clearInterval(timeInterval)
+  })
 </script>
 
 {#if $toast}
-  {#key "key"}
+  {#key $toast.id}
```

Comment:
This was actually correct, it's to avoid an animation when there is already a toast visible. Unless something changed?

Reply:
I have a weird behaviour with the "key" key where the content of 2 different toast is merged together. 

I have the first delay toast staying for 3 seconds followed by the "publish" toast with a 5 seconds timeout. If I don't have a uniq key, then both toast content are merged together until the 5 seconds timeout of the second toast.